solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -21,14 +21,20 @@ git clone https://github.com/Seagate/cortx-re && cd $PWD/cortx-re/solutions/comm
 
 - During tools installation, your are prompted to enter the AWS Access and Secret key. For more details, refer [AWS CLI Configuration](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config).
 
+- Execute below command to check the connectivity with AWS
+```
+aws sts get-caller-identity
+```
+
 - Modify `user.tfvars` file with your AWS details.
 ```
 os_version          = "<OS VERSION>"
 region              = "<AWS REGION>"
 security_group_cidr = "<YOUR PUBLIC IP CIDR>"
-key_name            = "<AWK KEY NAME>"
+pem_keyfile_name    = "<AWS KEY NAME>"
```

Comment:
This change needs to be also updated in single-node.tf  e.g. https://github.com/subhalaxmisa/cortx-re/blob/CORTX-32079/solutions/community-deploy/cloud/AWS/single-node.tf#L108
I guess we can keep a name as `key_name` but add an explanation in the document. 

Maybe create a table in https://github.com/Seagate/cortx-re/blob/main/solutions/community-deploy/cloud/AWS/README.md#install-and-setup-terraform-and-aws-cli with variable there details.

Reply:
Maybe create a table in https://github.com/Seagate/cortx-re/blob/main/solutions/community-deploy/cloud/AWS/README.md#install-and-setup-terraform-and-aws-cli with variable there details.

@shailesh-vaidya Can you please explain the above one?